frontend/src/Models/TransactionModel.ts

```diff
@@ -1,4 +1,5 @@
 // copied over from the transaction-frontend branch
+// because our code review is slooooooow
```

Comment:
Please leave these comments outside the code. It would be more efficient to write this in slack.

Reply:
This comment will be removed when merged to development (removing the duplicate `TransactionModel.ts`)
A more interesting discussion would be regarding the multiple model/interface definitions in a single file (`TransactionModel.ts`) - should i split them up?

Reply:
Gal ir gali buti vienam, bet reiketu informatyviau pavadint faila

Reply:
Btw butu gerai uzvesti koki /utils/ folderiuka visokiom shared funkcijom, kaip ta su datom or whatever

Reply:
Turi kokių pasiūlymų pavadinimui?
O dėl utėlių folderiuko, gal pasidarysim jei matysim kad dar reik kažkur naudot

Reply:
O gal, kita vertus, tai yra pakankamai geras pavadinimas